tests/modules/nf-core/beagle/beagle5/main.nf

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { BEAGLE_BEAGLE5 } from '../../../../../modules/nf-core/beagle/beagle5/main.nf'
+
+workflow test_beagle_beagle5 {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/beagle/target.22Jul22.46e.vcf.gz", checkIfExists: true)
+    ]
+    []
+    []
+    []
+    []
```

Comment:
Hi @nvnieuwk, thanks for the prompt review and comments. Instead of these empty lists I have now changed to
    genmap      = []
    exclsamples = []
    exclmarkers = []
I guess this would more explicitely show that these are optional arguments equaling to empty values in those specific cases. I saw this done in the e.g., gatk4/getpileupsummaries module. Would this be fine?

Reply:
Yes this is fine too :) I just thought it was weird you put some empty lists there.